src/test/java/com/google/api/codegen/testsrc/libraryproto/library_gapic.yaml

```diff
@@ -240,7 +240,7 @@ interfaces:
         - edition=123
         attributes:
         - parameter: shelf.name
-          sample_argument_name: name
+          sample_argument_name: shelf_name
```

Comment:
For brevity, I slightly lean toward s/`sample_argument_name`/`sample_argument`/, but with the same semantics. Thoughts? @beccasaurus